packages/btcindexer/src/index.ts

```diff
@@ -35,6 +36,21 @@ export default {
 		}
 	},
 
+	async queue(
+		batch: MessageBatch<BlockQueueMessage>,
+		env: Env,
+		_ctx: ExecutionContext,
+	): Promise<void> {
+		console.log(`Processing batch of ${batch.messages.length} messages from ${batch.queue}`);
+		const nbtcAddresses = await fetchNbtcAddresses(env.DB);
+		const nbtcAddressesMap = new Map<string, NbtcAddress>(
+			nbtcAddresses.map((addr) => [addr.btc_address, addr]),
```

Comment:
this is for all active/ inactive btc addresses? We should know if it's active or not. Maybe add TODO? Otherwise explain in the comment why we have a map.

Reply:
this are all addresses, we do not distinguish it. Added a todo for it